test/framework/config.go

```diff
@@ -254,10 +258,9 @@ var defaultConf = E2eConfig{
 	},
 	ZoneEgressApp:                     "kuma-egress",
 	ZoneIngressApp:                    "kuma-ingress",
-	UniversalE2ELogsPath:              path.Join(os.TempDir(), "e2e"),
-	CleanupLogsOnSuccess:              false,
 	KumaExperimentalSidecarContainers: false,
-	DebugDir:                          path.Join(os.TempDir(), "e2e-debug"),
+	DumpDir:                           path.Join("..", "..", "..", "build", "reports", "e2e-debug"),
```

Comment:
What is the "current working directory" here, will it change? Can we leave a comment here?